app/reactiveLayer/behaviorManager/conf/default.ini

```diff
@@ -2,7 +2,7 @@ use_ears        false
 
 [BEHAVIORS]
 
-behaviors	(tagging, pointing, moveObject, ask, followingOrder, recognitionOrder)
+behaviors	(tagging, pointing, moveObject)
 ttsOptions      "iCub"
 
 [followingOrder]
```

Comment:
Sorry, do we keep the option to ask "what did you learn from the arm babbling?" ? In that case I'll have to add this as a behavior (should be done quickly).